sky/server/common.py

```diff
@@ -179,15 +181,17 @@ def get_cookies_from_response(
     return cookies
 
 
-@annotations.lru_cache(scope='global')
-def get_server_url(host: Optional[str] = None) -> str:
-    endpoint = DEFAULT_SERVER_URL
-    if host is not None:
-        endpoint = f'http://{host}:46580'
-
+def get_server_url(host: Optional[str] = DEFAULT_SERVER_HOST) -> str:
```

Comment:
```suggestion
def get_server_url(host: str = DEFAULT_SERVER_HOST) -> str:
```
Can we make sure there aren't any usages of this that are depending on the `host=None` behavior?